.github/workflows/python-package.yml

```diff
@@ -24,40 +24,27 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Install pip
+      run: python -m pip install --upgrade pip
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install flake8
+        python -m pip install --upgrade setuptools wheel twine
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         if [ -f requirements-dev.txt ]; then pip install -r requirements-dev.txt; fi
-    - name: Lint with flake8
```

Comment:
All of this block of code is required for this project to be run and tested. Please ensure you have read the documentation properly before proceeding

Reply:
Can you please review this code is this correct or not ?

name: Python package

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

jobs:
  build:

    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
      matrix:
        python-version: [3.9, 3.11]

    steps:
    - uses: actions/checkout@v2

    - name: Set up Python ${{ matrix.python-version }}
      uses: actions/setup-python@v2
      with:
        python-version: ${{ matrix.python-version }}

    - name: Upgrade pip
      run: |
        python -m pip install --upgrade pip

    - name: Install dependencies
      run: |
        python -m pip install flake8
        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
        if [ -f requirements-dev.txt ]; then pip install -r requirements-dev.txt; fi

    - name: Lint with flake8
      run: |
        # TODO: stop the build if there are Python syntax errors or undefined names
        flake8 . --count --exit-zero --select=E9,F63,F7,F82 --show-source --statistics
        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics

    - name: Install package
      run: |
        python setup.py install

    - name: Download certificate and private key
      run: |
        wget "https://www.sistemasagiles.com.ar/soft/pyafipws/reingart2021.zip" -O reingart2019.zip
        unzip reingart2019.zip

    - name: Copy configuration files
      run: |
        sudo cp conf/rece.ini rece.ini
        sudo cp conf/wslum.ini wslum.ini
        sudo cp conf/wsremcarne.ini wsremcarne.ini
        sudo cp conf/wsltv.ini wsltv.ini
        sudo cp conf/wslsp.ini wslsp.ini

    - name: Install pip dependencies
      run: |
        python -m pip install --upgrade setuptools wheel twine

    - name: Build package
      run: |
        python setup.py sdist bdist_wheel

    - name: Publish distribution 📦 to PyPI
      uses: pypa/gh-action-pypi-publish@master
      with:
        user: __token__
        password: ${{ secrets.PYPI_API_TOKEN }}

    - name: Publish distribution 📦 to Test PyPI
      if: secrets.TEST_PYPI_API_TOKEN != ''
      uses: pypa/gh-action-pypi-publish@release/v1
      with:
        user: __token__
        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
        repository_url: https://test.pypi.org/legacy/

    - name: Test with pytest
      run: |
        pytest --html=report.html --self-contained-html

    - name: Upload test report
      uses: actions/upload-artifact@v2
      if: ${{ always() }}
      with:
        name: report.html
        path: report.html

    - name: Coverage Report
      run: |
        pytest --cov-config=.coveragerc --cov-report=html --cov=pyafipws tests/

    - name: Upload coverage report
      uses: actions/upload-artifact@v2
      if: ${{ always() }}
      with:
        name: Coverage_Report
        path: htmlcov

Reply:
I know it's your first opensource work
So it's good to always ask especially with issues like deleting code somewhere

Reply:
Ok Sir, I might have mistakenly deleted it but updated in the previous code can you please review it

Reply:
@baivab85 First, you need to sign-off the commits and then send a new commit with the changes requested by @HanslettTheDev so that he can review your work again. 
Don't worry if you have to send more commits until your work can be approved, that's the process and you gain practice in the meantime.
Thank you for your collaboration.

Reply:
@baivab85 Try to push the the github script in this same branch
I noticed some minor changes in it, Just push to this branch so the github runners can execute it so we can see the commit changes per line